source/common/http/conn_pool_base.cc

```diff
@@ -207,36 +207,13 @@ uint64_t MultiplexedActiveClientBase::maxStreamsPerConnection(uint64_t max_strea
   return (max_streams_config != 0) ? max_streams_config : DEFAULT_MAX_STREAMS;
 }
 
-MultiplexedActiveClientBase::MultiplexedActiveClientBase(HttpConnPoolImplBase& parent,
-                                                         uint32_t max_concurrent_streams,
-                                                         Stats::Counter& cx_total)
+MultiplexedActiveClientBase::MultiplexedActiveClientBase(
+    Envoy::Http::HttpConnPoolImplBase& parent, uint32_t effective_concurrent_streams,
```

Comment:
Out of curiosity, why the change from "HttpConnPoolImplBase" to "Envoy::Http::HttpConnPoolImplBase". This code is in the namespace Envoy::Http so I would have expected that to be redundant?

Reply:
oh, that was from the "deleted" constructor below - git diff weirdness.  fixed either way